participants/fbook.md

```diff
@@ -0,0 +1,70 @@
+# <fBook>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+**Our project name is fBook, I'm Joe, Allen in Slack.**
+
+## (1) Brief Project Description
+
+- Share online novels for free
+- Everyone can download and view
+
+## (2) Link to Project UI
+
+- [ui](https://github.com/Thchoonlophon/fbook/tree/master/ui)
+
+
+## (3) What does your application/UI do?
+
+- Provide downloads of novels
+- search with cid
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+**txt**
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+**free ebook**
```

Comment:
Can you clarify the source of these books and whether you have the appropriate license?